R/RDataParam.R

```diff
@@ -0,0 +1,70 @@
+#' @title Store `XcmsExperiment` object as .RData file
+#'
```

Comment:
in general i'm a bit confused by what documentation a single method/new class is supposed to have. I have not found an example similar to this.  I have mostly found generic function detailed documentation. So I'm not sure about the structure and level of detail here.

Reply:
Ok So I remove everything and just leave @title, @name, @description and the 2 @param ? Or I keep it as it is ? 
I will add a sentence that specify that it gets exported using the `[save()]`.

Reply:
Actually, you will need to add also a `@param fileName` (and all other parameters for `RDataParam`). I would however remove the *Other available `param` classes* part. For `@return` I would write:

```
@return for `RDataParam`: a `RDataParam` class. `storeResults` does not return anything but saves the object to a *RData* file.
```

Reply:
And to link to the other parameters/options you can add a

```
@family xcms result export formats
```

somewhere in the documentation. the same should then also be added for the `PlainTextParam` and roxygen will then take care of linking the two together.